.clusterfuzzlite/parser_fuzzer.cpp

```diff
@@ -0,0 +1,16 @@
+#include <fuzzer/FuzzedDataProvider.h>
```

Comment:
Copyright is missing.

Reply:
Added copyright to Openddl-parser authors, let me know if this works.